stingray/lombscargle.py

```diff
@@ -472,12 +472,12 @@ def __init__(
 def lscrossspectrum_from_lightcurve(
     lc1,
     lc2,
-    norm="none",
-    power_type="all",
-    fullspec=False,
-    min_freq=0,
-    max_freq=None,
-    method="fast",
+    norm,
```

Comment:
Why the choice of having just positional arguments here?

Reply:
I wanted to force the use of the classes as opposed to the light curves

Reply:
I don't think we need to force anything. Having a function is often more intuitive for some users, and I'd say it doesn't hurt to have a simple interface with reasonable default values.